src/pages/ImportData.js

```diff
@@ -77,40 +77,224 @@ const ResponseFragment = ({ title, snapshotResponses, responseIndex }) => {
   ])
 }
 
-const ImportData = () => {
+const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, url }) => div({ style: styles.card }, [
```

Comment:
When we migrate to typescript, will these change to using a fixed list of typed arguments? Using a property list is nice in the case where some arguments are omitted, but it doesn't work as well for typed arguments and isn't as useful if no arguments are omitted.

Reply:
Yes these will be a list of typed arguments eventually. (Potentially a typed object if we don't want to worry about argument order)